src/views/asset-edit-view/layout.tsx

```diff
@@ -32,33 +32,40 @@ export const AssetEditLayout = ({
   const [errorDescription, setErrorDescription] = useState<string | null>(null);
 
   useEffect(() => {
-    getAddressViaUprn(assetDetails.assetAddress.uprn)
-      .then((searchAddressResponse) => {
-        if (searchAddressResponse.addresses) {
-          setLlpgAddress(searchAddressResponse.addresses[0]);
-        }
-      })
-      .catch(() => {
-        setErrorHeading("Unable to retrieve address suggestion from the Local Gazetteer");
-        setErrorDescription(
-          "Please refresh the page and try again, otherwise you are still able to edit the blank fields manually.",
-        );
-        setShowError(true);
-      })
-      .finally(() => setLoading(false));
+    if (assetDetails.assetAddress.uprn) {
+      setLoading(true);
+      getAddressViaUprn(assetDetails.assetAddress.uprn)
+        .then((searchAddressResponse) => {
+          if (searchAddressResponse.addresses) {
+            setLlpgAddress(searchAddressResponse.addresses[0]);
+          }
+        })
+        .catch(() => {
+          setErrorHeading(
+            "Unable to retrieve address suggestion from the Local Gazetteer",
+          );
+          setErrorDescription(
+            "Please refresh the page and try again, otherwise you are still able to edit the blank fields manually.",
+          );
```

Comment:
This error is a bit wordy. How about just using the fist part: "Please refresh the page and try again"

Reply:
This was suggested by Sean P a while back, so I'm tempted to leave it as it is